addons/TBMod_arsenal/arsenal/fn_arsenalUSA.sqf

```diff
@@ -1065,6 +1065,12 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
             "rhsusf_spcs_ocp_sniper",
             "rhsusf_spcs_ucp_sniper",
 
+            // Munition
+            "rhsusf_mag_10Rnd_STD_50BMG_mk211",
```

Comment:
Nicht notwenig!
Wir schreiben keine Muni in die Rollen... dafür sind die Mengen da.... Spotter hat Zugriff auf die `_sniper_munition` Menge

Reply:
Da hast du vollkommen Recht! Muss übernächtet gewesen sein -.-

Reply:
wenn du ein Kommentar gemacht hast, gehe unten auf "Resolve conversation"... dann weiß ich es ist erledigt